src/app/applyContextMenu.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { clipboard, Menu, MenuItem } from 'electron'
+
+/**
+ * Enable context menu by right click
+ *
+ * @param {import('electron').BrowserWindow} browserWindow - Browser window
+ */
+export function applyContextMenu(browserWindow) {
+	browserWindow.webContents.on('context-menu', async (event, params) => {
+		const menu = new Menu()
+
+		// Add context actions for misspeling words and typos
+		const menuMisspellingItems = [
+			...params.dictionarySuggestions.map(suggestion => ({
+				label: suggestion,
+				click: () => browserWindow.webContents.replaceMisspelling(suggestion),
+			})),
+			{
+				label: 'Add to dictionary',
+				click: () => browserWindow.webContents.session.addWordToSpellCheckerDictionary(params.misspelledWord),
+			},
+			{ type: 'separator' },
+		]
```

Comment:
Maybe add a separator between `suggestions` and `Add to dictionary`? So it won't feel like one of the suggestions. It's also more common for context menus (at least in all apps I have looked at).